pydal/_load.py

```diff
@@ -5,10 +5,12 @@
     from collections import OrderedDict
 except:
     from .contrib.ordereddict import OrderedDict
+    
 try:
+# try external module
+    import simplejson as json
```

Comment:
simplejson is updated more frequently and is generally faster than the json module shipped with python. It's not going to hurt anybody if you leave that line, importing simplejson if it's found and fallbacking to json.